docs/user/login-alternatives.md

```diff
@@ -0,0 +1,36 @@
+# Kubeapps Login Form
+
+By default, when first visiting Kubeapps, a login form is shown for the user to introduce a personal token for a specific service account:
+
+![Dashboard Login](../img/dashboard-login.png)
+
+You can find more information about access control in Kubeapps in this [document](./access-control.md).
+
+However, it's possible to disable this form or delegate the authentication to an OIDC provider so Kubeapps users don't need to introduce a token in the login form.
+
+## Bypassing authentication
+
+Kubeapps expects an `Authorization` header that will be used to validate operations agains the Kubernetes API. This is usually set with the login form but if Kubeapps is exposed with an Ingress object, it's possible to hardcode a valid token in the Ingress configuration and automatically include it in all the requests.
+
+**NOTE**: This is not suitable for production since anyone with access to Kubeapps would be granted with the permissions associated with the hardcoded token.
```

Comment:
```suggestion
**NOTE**: This is not recommended in production since anyone with access to Kubeapps would be granted with the permissions associated with the hardcoded token.
```

I think "recommended" might be a better word to use here.